src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -18,13 +18,16 @@
 
 import os
 import re
+from typing import Callable, List, Optional, Union
 
 import numpy
 
+from transformers.modeling_tf_utils import TFPreTrainedModel
+from transformers.modeling_utils import PreTrainedModel
```

Comment:
I'd rather we not import those here without the correct framework safeguards, so I'd just put them in-between quotes rather than importing them

Reply:
Okay Sure